src/openrct2/interface/Screenshot.cpp

```diff
@@ -701,9 +702,9 @@ int32_t cmdline_for_screenshot(const char** argv, int32_t argc, ScreenshotOption
             if (customLocation)
             {
                 if (centreMapX)
-                    customX = (mapSize / 2) * 32 + 16;
```

Comment:
customX/customY should be refactored in another PR (making a note so that I don’t forget).